### PR TITLE
PR#2 Almost Finalized Portfolio 

### PR DESCRIPTION
**Pull request by cxfloeder**
The structure is finalized. The plan for the future is to add more CSS features for practice and to make the portfolio more visually appealing.

### CODE REVIEW

File: portfolio/src/main/webapp/Image_Gallery.html [hunk 1]
```diff
@@ -6,18 +6,43 @@
     <title>Charlie's Gallery</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <style>
```
Comment (ianjjohnson) [inline portfolio/src/main/webapp/Image_Gallery.html:9]: Can you put this style into a CSS file and reference it here?
(Maybe also put a comment there describing what that color is)

File: portfolio/src/main/webapp/LinkedIn_Info.html [hunk 2]
```diff
@@ -6,10 +6,18 @@
 	<title>Web Information</title>
 	<link rel="stylesheet" href="style.css">
 	<script src="script.js"></script>
+    <style>
+        body {
+             background-color: rgb(242, 242, 242);
+        }
+    </style>
```
Comment (ianjjohnson) [inline portfolio/src/main/webapp/LinkedIn_Info.html:13]: Then you can also reference that CSS class from here

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -6,11 +6,19 @@
 	<title>My Portfolio</title>
 	<link rel="stylesheet" href="style.css">
 	<script src="script.js"></script>
+    <style>
```
Comment (ianjjohnson) [inline portfolio/src/main/webapp/index.html:9]: Also here

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -29,3 +29,10 @@ function addRandomGreeting() {
     const greetingContainer = document.getElementById('greeting-container');
     greetingContainer.innerText = greeting;
 }
+
+/**
+ * Increments the value on times the button has been clicked
+ */
+function incrementCount() {
+
```
Comment (jakecarlson1) [inline portfolio/src/main/webapp/script.js:37]: Is this meant to be empty?

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -38,15 +46,15 @@ <h2>Quick Bio</h2>
             Center at college.
 		</p>
         <h2>Random Facts</h2>
-        <p>
+        <ul>
             <! ––  make a list here. ––>
```
Comment (ianjjohnson) [inline portfolio/src/main/webapp/index.html:50]: You can probably delete this comment

File: portfolio/src/main/webapp/Image_Gallery.html [hunk 6]
```diff
@@ -6,18 +6,43 @@
...
+        body {
+             background-color: rgb(242, 242, 242);
+        }
```
Comment (jakecarlson1) [inline portfolio/src/main/webapp/Image_Gallery.html:12]: Thoughts on moving this to the style sheet?

File: portfolio/src/main/webapp/script.js [hunk 7]
```diff
@@ -29,3 +29,10 @@ function addRandomGreeting() {
     const greetingContainer = document.getElementById('greeting-container');
     greetingContainer.innerText = greeting;
 }
+
+/**
```
Comment (ianjjohnson) [inline portfolio/src/main/webapp/script.js:33]: Consider removing this empty function

(Or write it and make it not empty :) )

File: portfolio/src/main/webapp/style.css [hunk 8]
```diff
@@ -8,10 +24,69 @@
     margin-top: 20px;
 }
 
+/**
+ this is where the Image_Gallery.html specific CSS stuff is
```
Comment (ianjjohnson) [inline portfolio/src/main/webapp/style.css:28]: Why not make a separate file for CSS related to the image gallery?

File: portfolio/src/main/webapp/style.css [hunk 9]
```diff
@@ -8,10 +24,69 @@
...
+
+#picture:hover {
+  border: thin solid black;
+}
+
+#caption {
+    padding: 15px;
+    text-align: center;
+}
+/**
```
Comment (ianjjohnson) [inline portfolio/src/main/webapp/style.css:57]: Same here (make a separate CSS file)

File: portfolio/src/main/webapp/Image_Gallery.html [hunk 10]
```diff
@@ -6,18 +6,43 @@
...
-            <a href="new_images/IMG_0085.JPG"><img src="new_images/IMG_0085.JPG"/></a>
-            <a href="new_images/IMG_0222.JPG"><img src="new_images/IMG_0222.JPG"/></a>
+        </div>
+        <div id="picture">
+            <a href="new_images/selfie.jpg"><img src="new_images/selfie.jpg"></a>
+            <div class="desc"><center>Selfie in front of NE capital building</center></div>
+        </div>
+        <div id="picture">
+            <a href="new_images/IMG_0085.JPG"><img src="new_images/IMG_0085.JPG"></a>
+            <div class="desc"><center>Selfie in front of a sick McLaren</center></div>
```
Comment (jakecarlson1) [inline portfolio/src/main/webapp/Image_Gallery.html:27]: This is a sick McLaren. (No action needed)

**Review by ianstenbit — COMMENTED** [review]
Nice additions! Looking forward to seeing how this looks once it gets deployed

File: portfolio/src/main/webapp/General.css [hunk 11]
```diff
@@ -0,0 +1,64 @@
+/**
```
Comment (ianjjohnson) [inline portfolio/src/main/webapp/General.css:1]: nit: this multi-line comment can be on a single line

File: portfolio/src/main/webapp/General.css [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+.previous {
+    background-color: rgb(177, 177, 177);
+    color: black;
+}
+
+.round {
+    border-radius: 50%;
+}
+
+/**
```
Comment (ianjjohnson) [inline portfolio/src/main/webapp/General.css:23]: Same for this comment (also capitalize the start)

File: portfolio/src/main/webapp/Home.css [hunk 13]
```diff
@@ -0,0 +1,21 @@
+/**
```
Comment (ianjjohnson) [inline portfolio/src/main/webapp/Home.css:1]: Also here

File: portfolio/src/main/webapp/Home.html [hunk 14]
```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>My Portfolio</title>
+	<link rel="stylesheet" href="Home.css">
+	<link rel="stylesheet" href="General.css">
```
Comment (ianjjohnson) [inline portfolio/src/main/webapp/Home.html:8]: Maybe General.css should be named common.css?
Also I think the convention is to use lowercase filenames for CSS files

File: portfolio/src/main/webapp/Image_Gallery.css [hunk 15]
```diff
@@ -0,0 +1,38 @@
+/**
+ this is where the Image_Gallery.html specific CSS stuff is
```
Comment (ianjjohnson) [inline portfolio/src/main/webapp/Image_Gallery.css:2]: Can also be a single-line comment

File: portfolio/src/main/webapp/Image_Gallery.html [hunk 16]
```diff
@@ -1,23 +1,65 @@
...
+	<div id="gallery_body">
+		<div class="drop_menu">
+			<button class="menu_button">Page Menu</button>
+			<div class="dropdown_content">
+				<a href="Home.html">Home</a>
+				<a href="Image_Gallery.html">Images</a>
+				<a href="Network_Info.html">Network Info</a>
+			</div>
+		</div>
+		<a href="Home.html" class="padding previous round">&#8249;</a>
```
Comment (ianjjohnson) [inline portfolio/src/main/webapp/Image_Gallery.html:22]: What does the &#8249 mean here?

**Review by ianstenbit — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/Image_Gallery.html (hunk 16 above)
Comment (cxfloeder) [reply portfolio/src/main/webapp/Image_Gallery.html:22]: It's a left character entity.

File: portfolio/src/main/webapp/Network_Info.html [hunk 17]
```diff
@@ -0,0 +1,44 @@
...
+			<center>Networking Info</center>
+		</h1>
+		<h2>LinkedIn</h2>
+		<p>
+			<a href="https://www.linkedin.com/in/charles-floeder-878a12192/2">
+				LinkedIn URL
+			</a>
+		</p>
+		<h2>GitHub</h2>
+		<p>username: cxfloeder</p>
```
Comment (jakecarlson1) [inline portfolio/src/main/webapp/Network_Info.html:33]: You might want to link to your GitHub page. That will make it easier for users to find the information they are looking for.

File: portfolio/src/main/webapp/Image_Gallery.html (hunk 16 above)
Comment (ianjjohnson) [reply portfolio/src/main/webapp/Image_Gallery.html:22]: Could you add a comment explaining what it is in the code?

File: portfolio/src/main/webapp/General.css (hunk 11 above)
Comment (cxfloeder) [reply portfolio/src/main/webapp/General.css]: fixed

File: portfolio/src/main/webapp/General.css (hunk 12 above)
Comment (cxfloeder) [reply portfolio/src/main/webapp/General.css]: fixed

File: portfolio/src/main/webapp/Home.css (hunk 13 above)
Comment (cxfloeder) [reply portfolio/src/main/webapp/Home.css]: fixed

File: portfolio/src/main/webapp/Home.html (hunk 14 above)
Comment (cxfloeder) [reply portfolio/src/main/webapp/Home.html]: fixed

File: portfolio/src/main/webapp/Image_Gallery.css (hunk 15 above)
Comment (cxfloeder) [reply portfolio/src/main/webapp/Image_Gallery.css]: fixed

File: portfolio/src/main/webapp/Network_Info.html (hunk 17 above)
Comment (cxfloeder) [reply portfolio/src/main/webapp/Network_Info.html:33]: fixed

**Review by jakecarlson1 — APPROVED** [review]
LGTM

File: portfolio/src/main/webapp/Image_Gallery.html (hunk 16 above)
Comment (ianjjohnson) [reply portfolio/src/main/webapp/Image_Gallery.html:22]: Bumping this thread. Could you add a comment to the code about this?

**Review by ianstenbit — APPROVED** [review]

**Pull request merged**